README.md

```diff
@@ -1 +1,168 @@
-# ruby_catalog
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <h3><b>[Catalog of my things] is a project at Microverse. It's an app that keeps records of books, games, and music album.
```

Comment:
- [ ] I think you should remove the project description from here and add it under the description section of your readme file to make it more professional :+1:

Reply:
Done. Thanks